run-reports.sh

```diff
@@ -1,2 +1,2 @@
-ruby /app/script/create_analytics_report.rb -i ga:84655146 -a /app/config/google_auth.json -s 2017-11-01 -e 2017-11-02 -o /app
-cat /app/output_2017-11-01_2017-11-02.csv
```

Comment:
1.  Do we need to cat it?
1.  I think  it's a safer bet to run `/ruby/app/script...` right? Maybe I'm missing something.

Reply:
You're right. Going to remove `cat` statement.